### PR TITLE
Add security rationale; fixes #39

### LINKED ISSUE #39: link to "security rationale" is broken
The link to the "security rationale" in the readme is broken

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Add text for security rationale. I had been written this before,
but I accidentally didn't do a "git add" :-).
My thanks to Greg K-H for noticing & reporting this!

Fixes #39.

Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

### CODE REVIEW

**Comment by david-a-wheeler** [comment]
@SecurityCRob @xcorail @jnaulty - I forgot to do "git add" before, sorry. Hopefully this helps! Seem okay?

File: security-rationale.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+However, this would be true for any shop, and these are reputable shops.
+
+## Project worries personal information may be shared with OpenSSF or someone who subverts the OpenSSF
+
+The OpenSSF never receives the actual names or addresses
+of the recipients. The OpenSSF is asking projects to report back
+summary metrics. GitHub will learn the GitHub accounts of those who
+use the validation codes, but GitHub will only share totals
+back to the OpenSSF.
+We this it's reasonable that someone who is willing to create an
```
Comment (xcorail) [inline security-rationale.md]:
```suggestion
We think it's reasonable that someone who is willing to create an
```

**Review by xcorail — APPROVED** [review]
Small typo

**Comment by david-a-wheeler** [comment]
Awesome, thanks for the typo fix!

**Pull request merged**